webhooks-extension/src/components/WebhookCreate/tests/WebhookCreate.test.js

```diff
@@ -190,6 +199,25 @@ describe('drop downs should be enabled when a namespace is selected', () => {
     await wait(() => expect(document.getElementById("serviceAccounts").getElementsByClassName("bx--list-box__field").item(0).hasAttribute("disabled")).toBe(false))
 
     });
+  it('service accounts dropdown disabled due to fetch error', async () => {
+    jest.spyOn(API, 'getSecrets').mockImplementation(() => Promise.resolve(secretsResponseMock));
+    const { getByText } = renderWithRouter(
+      <WebhookCreate
+        match={{}}
+        namespaces={namespaces}
+        pipelines={pipelines}
+        fetchPipelines={() => {}}
+        fetchServiceAccounts={() => {}}
+        isFetchingPipelines={false}
+        isFetchingServiceAccounts={false}
+        serviceAccounts={serviceAccounts}
+        getServiceAccountsErrorMessage={"Error fetching service accounts."}
+        setShowNotificationOnTable={() => {}}
+      />
+    );
+    await wait(() => expect(document.getElementById("pipeline").getElementsByClassName("bx--list-box__field").item(0).hasAttribute("disabled")).toBe(true))
```

Comment:
can we avoid these native DOM functions and instead use the API provided by react-testing-library based on the content visible to the user?

Reply:
i can go ahead & do it but most of the tests were already using DOM fns. It might make the PR really big

Reply:
ok let's just update these new tests, the others can be addressed later

Reply:
ok let's just update these new tests, the others can be addressed later